README.md

```diff
@@ -62,6 +62,8 @@ function NewComponent(props) {
 }
 ```
 
+Some things in the Design System, such as the universal focus styling and the box-model the DS uses, are namespaced to the `.nypl-ds` class. To include those in your app, add `.nypl-ds` to whichever wrapper tag makes sense in your application.
```

Comment:
> to whichever wrapper tag makes sense 
Can we be more specific than this?  Something like, "to include all `.nypl-ds` styles, add `.nypl-ds` to the html tag"?  

I think if people want to pick and choose they can actually dig into it, but giving people an easy quick start would be handy

Reply:
Added some more specific langugage